Add license information to main page of OTJ

josephsnyder opened the pull request:
Add the license information to the template that is used to display
items on the main page of the OTJ.  Add the necessary CSS to emphasize
the license information.

josephsnyder committed:
Add license information to main page of OTJ

Add the license information to the template that is used to display
items on the main page of the OTJ.  Add the necessary CSS to emphasize
the license information.

BetsyMcPhail commented:
I'm wondering if it would be more clear which submission the license info belonged to if it was in the top-right corner (above or below the "download/viewed" text)? 

Otherwise, the information is displaying as expected

josephsnyder committed:
Re-arrange License information to top right corner

Move the license information to the top right corner, underneath the
download information.  This should prevent confusion about which
submission the information belongs with.
